SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,48 @@
+uri "http://kdl.ge.com/AM-CDM-manufacturingProcess" alias AM-CDM-manufacturingProcess version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-process".
+import "http://kdl.ge.com/AM-CDM-system".
+import "http://kdl.ge.com/AM-CDM-material".
+
+ManufacturingProcessStep (note "Planned process in which input material is transformed or modified") is a type of PlannedProcessStep,
+	described by name (note 'Human readable name which designates this process step.') with a single value of type string,
+	described by processStepSpecification with a single value of type ManufacturingSpecification,
+	described by processMachine (note "The machine used to transform the input material.") with a single value of type System,
+	described by controlSoftware (note "Software used to control the process.") with values of type Software,
+	described by consumables (note "Consumables used during this process step.") with values of type Consumables,
+	described by materialInputs ("Material inputs for this process step") with values of type MaterialInputs,
+	described by processSimulation (note "Reference to Simulation records associated with this processStep.") with values of type ProcessSimulation.
```

Comment:
does materialOutput need to be here? It looks like the only way to have an output at this time is for it to be a ManufacturingCreationProcessStep which has a materialOutput